pkg/azuredisk/nodeserver.go

```diff
@@ -491,9 +519,21 @@ func (d *Driver) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolume
 	if err != nil {
 		return nil, status.Errorf(codes.NotFound, "failed to determine device path for volumePath [%v]: %v", volumePath, err)
 	}
+	if !isBlock {
```

Comment:
I don't see the changes below in `nodeserver_v2` or the `NodeProvisioner`.

Reply:
Fixed